index.js

```diff
@@ -6,17 +6,18 @@
 module.exports = isError;
 
 /**
- * Test whether `V` is error object.
+ * Test whether `value` is error object.
  *
- * @param {*} V
+ * @param {*} value
  * @returns {boolean}
  */
 
-function isError(V) {
-  var tagName = Object.prototype.toString.call(V);
+function isError(value) {
+  var tag = Object.prototype.toString.call(value);
   return (
-    tagName === '[object Error]' ||
-    tagName === '[object Exception]' ||
-    V instanceof Error
+    tag === '[object Error]' ||
```

Comment:
http://archive.oreilly.com/pub/a/server-administration/excerpts/even-faster-websites/writing-efficient-javascript.html

    Use the if statement when:

        There are no more than two discrete values for which to test.

        There are a large number of values that can be easily separated into ranges.

    Use the switch statement when:

        There are more than two but fewer than 10 discrete values for which to test.

        There are no ranges for conditions because the values are nonlinear.

    Use array lookup when:

        There are more than 10 values for which to test.

        The results of the conditions are single values rather than a number of actions to be taken.